proto/mixer.proto

```diff
@@ -237,6 +237,37 @@ message GetStatsResponse {
   string payload = 2;
 }
 
+// GetStat is a newer version of GetStats API, that use protobuf type as return
+// field instead of a "payload" of json string.
+// TODO(shifucun): Deprecate GetStats() once all clients are switched to use
+// this API.
+// Request message for GetStat.
+message GetStatRequest {
+  // The dcids of the place.
+  repeated string places = 1;
```

Comment:
To confirm, are other places calling it dcids?

Reply:
All the new GetStat* API uses "places". Some of the old one like GetPopulation use "dcids". We decided to use "places" uniformly in the new API, since dcid is too general as there are statvar dcids in the request too.